CONTRIBUTING.md

```
@@ -76,6 +76,35 @@ make &&
     bin/tssc deploy --help
 ```
 
+## Debugging `tssc mcp-server`
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)